### PR TITLE
add button to send emails to group as bcc

### PR DESCRIPTION
**Pull request by caplod**
To comply with GDPR mass emails are supposed to be send as bcc. 
This commit adds a button to send emails to a group as bcc.

### CODE REVIEW

**Comment by caplod** [comment]
contacts

File: l10n/de.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ OC.L10N.register(
     "Address books" : "Adressbücher",
     "Download" : "Herunterladen",
     "Send email" : "E-Mail senden",
+    "Send email as bcc" : "E-Mail als bcc senden",
```
Comment (ChristophWurst) [inline l10n/de.js:34]: translations are handled via Transifex. Please undo this change. the new string will be sent to and pulled from transifex with a daily cron job

File: l10n/de.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "Address books" : "Adressbücher",
     "Download" : "Herunterladen",
     "Send email" : "E-Mail senden",
+    "Send email as bcc" : "E-Mail als bcc senden",
```
Comment (ChristophWurst) [inline l10n/de.json:32]: same

File: src/components/AppNavigation/GroupNavigationItem.vue [hunk 3]
```diff
@@ -47,6 +47,12 @@
 				</template>
 				{{ t('contacts', 'Send email') }}
 			</ActionButton>
+			<ActionButton @click="emailGroup(group, 'bcc')">
+				<template #icon>
+					<IconEmail :size="20" />
+				</template>
+				{{ t('contacts', 'Send email as bcc') }}
```
Comment (ChristophWurst) [inline src/components/AppNavigation/GroupNavigationItem.vue:54]:
```suggestion
				{{ t('contacts', 'Send email as BCC') }}
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: src/components/AppNavigation/GroupNavigationItem.vue (hunk 3 above)
Comment (caplod) [reply src/components/AppNavigation/GroupNavigationItem.vue]: Ok. done

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Then squash and we are ready for merge

**Comment by ChristophWurst** [comment]
There is an unrelated commit on your branch.

**Comment by caplod** [comment]
> There is an unrelated commit on your branch.

Fixed it. Sorry I am new to rebasing

**Comment by ChristophWurst** [comment]
> Sorry I am new to rebasing

https://classicprogrammerpaintings.com/post/142586036029/junior-programmer-learns-git-rebase :D

No worries. You just managed to fix the situation. You got this.
